hypervisor/src/kvm/mod.rs

```diff
@@ -578,17 +588,16 @@ impl cpu::Vcpu for KvmVcpu {
     /// let state = vcpu.cpu_state().unwrap();
     ///
     fn cpu_state(&self) -> cpu::Result<CpuState> {
-        let mut msrs = boot_msr_entries();
-        self.get_msrs(&mut msrs)?;
-
-        let vcpu_events = self.get_vcpu_events()?;
+        let mp_state = self.get_mp_state()?;
```

Comment:
In the commit message you mention that state can change as part of getting the other state. How can this happen if the vCPU is not running?

Reply:
The vCPU is technically "paused", but the states maintained by KVM can be changed even when the vCPU is not scheduled to run.

Reply:
That's a bit surprising so definitely make sure there is a comment about that.